detox/src/Detox.test.js

```diff
@@ -95,12 +96,21 @@ describe('Detox', () => {
     expect(() => detox.cleanup()).not.toThrowError();
   });
 
+  it(`Calling detox.init() twice returns the same promise`, async () => {
```

Comment:
I can't really explain why I specifically would want a single promise to be returned, but anyway, we use each instance of Detox class only once, so why not? I mean, calling the global `detox.init` works with `DetoxExportWrapper` which is responsible for the creation of Detox instances. Hence, technically, each Detox **instance** is `init`-ed only once, so I don't feel like this change is a mistake.

Reply:
Ah, right. And then, I am able to understand that init is actually happening till now, so I can reject init promise.